internal/commands/scan.go

```diff
@@ -189,14 +191,15 @@ func NewScanCommand(
 	scaRealtimeCmd := scarealtime.NewScaRealtimeCommand(scaRealTimeWrapper)
 
 	addFormatFlagToMultipleCommands(
-		[]*cobra.Command{listScansCmd, showScanCmd, workflowScanCmd},
-		printer.FormatTable, printer.FormatList, printer.FormatJSON,
+		[]*cobra.Command{scanLightweightCmd, listScansCmd, showScanCmd, workflowScanCmd},
+		printer.FormatJSON, printer.FormatTable, printer.FormatList, printer.FormatJSON,
```

Comment:
```suggestion
		printer.FormatTable, printer.FormatList, printer.FormatJSON,
```
duplicate of printer.FormatJSON